recipes/openff-models/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "openff-models" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openff-models/archive/{{ version }}.tar.gz
+  sha256: 0b7a8585c8c5235a0ab17faef4ebb731bd0538b148afc5375b2ce9b02282f555
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - pydantic <2.0.0a0
+    - openff-units
+    - openff-utilities
+
+test:
+  requires:
+    - setuptools
```

Comment:
Why is `setuptools` listed as a requirement for tests?

Reply:
Not sure; probably not needed